### PR TITLE
add indonesian translation

### PR DESCRIPTION
**Pull request by atdx**

### CODE REVIEW

File: i18n/id.yaml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+- id: noposts_warning_description
+  translation:
+    "Setelah kamu memposting sesuatu ke dalam folder manapun di folder <b>content</b>, Postingan itu akan muncul disini. Hanya satu \
+    bagian (dengan postingan terbanyak) yang akan ditampilkan di beranda secara default."
+
+- id: noposts_warning_tip
+  translation:
+    "<b>Tip:</b> Kamu dapat menampilkan sebanyak mungkin bagian dengan parameter konfigurasi \
+    <b><a href=\"https://gohugo.io/functions/where/#mainsections\" rel=\"nofollow noopener\" target=\"_blank\">mainSections</a></b> \ ."
```
Comment (Vimux) [inline i18n/id.yaml:93]:
```suggestion
    "<b>Tip:</b> Kamu dapat menampilkan sebanyak mungkin bagian dengan parameter konfigurasi \
    <b><a href=\"https://gohugo.io/functions/where/#mainsections\" rel=\"nofollow noopener\" target=\"_blank\">mainSections</a></b>."
```

Extra line splitter here.

File: i18n/id.yaml [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  translation: "Beranda"
+
+# No posts empty state
+- id: noposts_warning_title
+  translation: "Kamu belum memiliki postingan apa pun!"
+
+- id: noposts_warning_description
+  translation:
+    "Setelah kamu memposting sesuatu ke dalam folder manapun di folder <b>content</b>, Postingan itu akan muncul disini. Hanya satu \
+    bagian (dengan postingan terbanyak) yang akan ditampilkan di beranda secara default."
```
Comment (Vimux) [inline i18n/id.yaml:88]:
```suggestion
    "Setelah kamu memposting sesuatu ke dalam folder manapun di folder <b>content</b>, Postingan itu akan muncul \
    disini. Hanya satu bagian (dengan postingan terbanyak) yang akan ditampilkan di beranda secara default."
```

For most of the translation file, we are targeting at 120 characters width line, except for `noposts_warning_tip`.

**Review by Vimux — CHANGES_REQUESTED** [review]
Need two small corrections.

**Comment by Vimux** [comment]
Thanks!

**Comment by atdx** [comment]
> Thanks!

Yes 👍

**Pull request merged**